app/lib/core/Search/SearchEngine.php

```diff
@@ -548,16 +531,15 @@ private function _rewriteTerm($po_term, $pb_sign) {
 		if (in_array($va_tmp2[1], array('preferred_labels', 'nonpreferred_labels'))) {
 			if ($t_instance = $this->opo_datamodel->getInstanceByTableName($va_tmp2[0], true)) {
 				if (method_exists($t_instance, "getLabelTableName")) {
-					return array(
-						'terms' => array(new Zend_Search_Lucene_Index_Term($po_term->getTerm()->text, $t_instance->getLabelTableName().'.'.((isset($va_tmp2[2]) && $va_tmp2[2]) ? $va_tmp2[2] : $t_instance->getLabelDisplayField()).($va_tmp[1] ? '/'.$va_tmp[1] : ''))),
-						'signs' => array($pb_sign),
-						'options' => array()
+					return new Zend_Search_Lucene_Search_Query_MultiTerm(
+						array(new Zend_Search_Lucene_Index_Term($po_term->getTerm()->text, $t_instance->getLabelTableName().'.'.((isset($va_tmp2[2]) && $va_tmp2[2]) ? $va_tmp2[2] : $t_instance->getLabelDisplayField()).($va_tmp[1] ? '/'.$va_tmp[1] : ''))),
+						array($pb_sign)
 					);
 				}
 			}
 		}
 		
-		return array('terms' => array($po_term->getTerm()), 'signs' => array($pb_sign), 'options' => array());
+		return new Zend_Search_Lucene_Search_Query_MultiTerm(array($po_term->getTerm()), array($pb_sign));
```

Comment:
Does this always have to be a `MultiTerm`? I'm just wondering whether the phpdoc should specify the parent class / interface?